docs/core/rid-catalog.md

```diff
@@ -102,14 +102,14 @@ Only common values are listed. For the latest and complete version, see the [run
   - `win10-arm`
   - `win10-arm64`
 
-See [.NET Core dependencies and requirements](install/dependencies.md?tabs=netcore30&pivots=os-windows) for more information.
+For more information, see [.NET Core dependencies and requirements](install/dependencies.md?tabs=netcore30&pivots=os-windows).
 
 ## Linux RIDs
 
 Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/runtime/blob/master/src/libraries/pkg/Microsoft.NETCore.Platforms/runtime.json) file on the `dotnet/runtime` repository. Devices running a distribution not listed below may work with one of the Portable RIDs. For example, Raspberry Pi devices running a Linux distribution not listed can be targeted with `linux-arm`.
 
 - Portable (.NET Core 2.0 or later versions)
-  - `linux-x64` (Most desktop distributions like CentOS, Debian, Fedora, Ubuntu and derivatives)
+  - `linux-x64` (Most desktop distributions like CentOS, Debian, Fedora, Ubuntu, and derivatives)
```

Comment:
doesn't derivatives, in this case, apply to Ubuntu?

Reply:
I don't think so, but if it does, we need to add "and Ubuntu and derivatives". Would @Thraka know?

Reply:
let's go with your suggestion since there are derivatives from the other distros too

Reply:
Correct. It's NOT Ubuntu's derivatives, but ANY derivative.